microcosm_postgres/temporary/factories.py

```diff
@@ -0,0 +1,57 @@
+"""
+Temporary table support.
+
+"""
+from types import MethodType
+
+from microcosm_postgres.temporary.methods import (
+    insert_many,
+    select_from,
+    upsert_into,
+)
+
+
+def create_temporary_table(from_table, name=None, on_commit=None):
+    """
+    Create a new temporary table from another table.
+
+    """
+    from_table = from_table.__table__ if hasattr(from_table, "__table__") else from_table
```

Comment:
This line could probably use a explanatory comment. I'm guessing that it has to do with `from_table` being a model class (or declarative base) vs a set of columns (e.g. the result of a `select`)?